librus/__init__.py

```diff
@@ -172,6 +172,28 @@ def schedule(self):
         lessons.sort(key=(lambda lesson: (lesson.day, lesson.index)))
         return lessons
 
+    def list_messages(self, get_content=False):
+        """
+        Gets messages (AKA 'wiadomości')
+        """
+        response = self._html_session.get(url='https://synergia.librus.pl/wiadomosci')
+
+        for row in response.html.find('.line0') + response.html.find('.line1'):
+            cells = row.find('td')
+            message = Message(
+                sender=cells[2].text,
+                subject=cells[3].text,
+                sent_at=datetime.datetime.strptime(cells[4].text, '%Y-%m-%d %H:%M:%S')
+            )
+            if get_content:
+                href = cells[3].find('a')[0].attrs['href']
+                url = 'https://synergia.librus.pl' + href
+                content = self._html_session.get(url=url).html.find('.container-message-content')[0]
+                message.set_content(content.text)
```

Comment:
is there any particular advantage of `set_content` vs `content = ` ?

like 

```suggestion
                message.content = content.text
```

?

Reply:
No reason, I like your way of keeping it simple!